Allow users to specify multiple languages to export

t1sh0o opened the pull request:
Users can pass a comma-separated list of languages to export the translations in.
This way they might run the command only once for all languages instead of once for each language.

Backward compatibility is preserved and the packages will work with one language as before.

t1sh0o committed:
Allow the command to accept multriple languages separated with comas

t1sh0o committed:
Document the feature for calling the command with multiple languages

kkomelin commented:
@t1sh0o Thank you Tihomir. I think this one is a really good idea. Please give me some time to test the change and merge it.

kkomelin committed:
Updated README

Improved the description and example of usage of the multiple language parameter. 
(English is a default language by default, so I removed it from the list)

kkomelin committed:
Updated README

Minor language change.

kkomelin committed:
Updated the lang param info

kkomelin commented:
@t1sh0o I've slightly updated the README and lang parameter help info but overall the PR works fine. Thanks. Going to merge it...
